### PR TITLE
Set user-agent and more reasonable artifact version

### PR DESCRIPTION
**Pull request by ctreatma**

### CODE REVIEW

File: spec/oas3.config.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
    "groupId": "com.equinix",
    "artifactId": "equinix-openapi-metal",
-   "artifactVersion": "1.0.0",
+   "artifactVersion": "0.1.0",
+   "httpUserAgent": "metal-java/0.1.0",
```
Comment (displague) [inline spec/oas3.config.json:5]: Presuming these were the only changes needed and the rest of the changes were generated, is there any reason this is still in draft? Looking to make additional improvements?

**Comment by ctreatma** [comment]
@bnallapeta it looks like we need to move the ProjectAPIKeyOperator so that it doesn't get deleted by the GitHub Action that automatically regenerates code for PRs: https://github.com/equinix-labs/metal-java/pull/92/files#diff-61b5c510cd217733ff1543e7fbf29e6425077cde90a6245fa4fe4d9384f43b49

Could you open a PR to copy or move that file into the [`internal` directory](https://github.com/equinix-labs/metal-java/tree/main/internal)?

File: spec/oas3.config.json (hunk 1 above)
Comment (ctreatma) [reply spec/oas3.config.json:5]: #93 should be merged first, and then this can be rebased and merged

Comment (ctreatma) [reply spec/oas3.config.json:5]: #93 has been merged, so this is good to go

**Review by displague — APPROVED** [review]

**Pull request merged**